commands.go

```diff
@@ -29,10 +29,6 @@ type AppendEntriesResponse struct {
 
 	// We may not succeed if we have a conflicting entry
 	Success bool
-
-	// There are scenarios where this request didn't succeed
-	// but there's no need to wait/back-off the next attempt.
-	NoRetryBackoff bool
```

Comment:
@rogpeppe we've got a number of non-compatible changes in this version of the library already, though they are generally pretty easy to deal with (it took about 2 hours to get Consul building and booting again). We intend to keep this in a branch and once it's done we will message the community and give a little bit of time for people to adapt to the changes before we take it to master. Whenever we can make something safer (like ID and addresses as separate types) or simpler we are breaking things. We've also got a story for how folks can interoperate with newer versions of the library during the transition - https://github.com/hashicorp/raft/blob/library-v2-stage-one/config.go#L10-L115.